src/components/SearchResults.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "../styles/search-results.css";
+
+const SearchResults = ({ searchResults }) => {
+    console.log(searchResults, "renders search results")
+    if (!searchResults.length) {
+        return <p className="no-results">Sorry, no results found. Try again. </p>
+    } else {
+    return (
+        <>
+        <div className="search-results">
+            {searchResults.map((image) => {
+                console.log(image, "image")
+            return <img className="search-results__image"
+                    src={image}
+                    alt="spaceImage" />
```

Comment:
This alt tag isnt all that helpful. Their purpose is to describe what an image is for people using a screen reader, in this case all the images will have this generic label that doesnt help describe anything really. A good next step for future development would be to see if you can get the alt tag for each image from the API and attach it to its image

Also, you are getting an error for not adding a key to each element in the map array here. Ideally an ID unique to each image would be returned from the API and put in here, but you can just use the iteration of the loop instead as a backup, eg:
```
{searchResults.map((image, id) => {
    return <img className="search-results__image"
        src={image}
        key={id}
        alt="spaceImage" />
    })}
```